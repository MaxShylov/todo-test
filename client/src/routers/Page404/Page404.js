import React from 'react';


const Page404 = () => <div className="Page404">Page 404</div>;


export default Page404;
