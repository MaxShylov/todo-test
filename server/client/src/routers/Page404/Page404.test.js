import React from 'react';
import { shallow } from 'enzyme'

import Page404 from './Page404.js'


describe('Page404', () => {

  describe('initial', () => {
    const wrapper = shallow(<Page404 />);

    it('renders properly', () => expect(wrapper).toMatchSnapshot());
  });

});
