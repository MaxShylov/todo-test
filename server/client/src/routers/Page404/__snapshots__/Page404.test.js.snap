// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404 initial renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Page404 />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Page 404",
      "className": "Page404",
    },
    "ref": null,
    "rendered": "Page 404",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Page 404",
        "className": "Page404",
      },
      "ref": null,
      "rendered": "Page 404",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
